mvp.feature

```diff
@@ -656,16 +663,52 @@ Feature: MVP features for NVA
     And they see a Delete button
 
   @278
-  Scenario: Creator publishes Publication
-    Given Creator sees Publication Channel Register information for a Journal
+  Scenario: Registrator publishes Publication
+    Given Registrator sees Publication Channel Register information for a Journal
     And they select values for uploaded files
     And they are on the Submission tab
     When they click Publish
     Then they are redirected to the Publication public page
     And they see the Publication confirmation message
 
+  @881
+  Scenario: View Public Page for publication
+    Given the user has opened NVA
+    And see the Start Page or Public Profile
+    When they click to view a publication
+    Then the public page for publication is opened
+    And the page fields for
+      | Title                  |
+      | Abstract               |
+      | NPI                    |
+      | Keywords               |
+      | Publication date       |
+      | Primary language       |
+      | Projects               |
+      | Publication type       |
+      | Publication subtype    |
+      | Values per sybtype     |
+      | Clickable Contributors |
+      | Clickable emails       |
+      | Affiliations           |
+      | File download link     |
+      | ORCID                  |
+      | DOI link               |
+      | Licence                |
+      | Licence link           |
+  # each licence must have some documentation a web page
+  # https://creativecommons.org/licenses/by/4.0/deed.no
+  # https://dlr.unit.no/licenses/ntnu-alle-rettigheter-forbeholdt-forfatter
+
+  @1236
+  Scenario: A user navigates to the landing page with a draft DOI
+    Given that a user navigates to the landing page
+    And the resource is published
+    And the resource has a draft DOI
+    Then the landing page is displayed without the draft DOI
```

Comment:
Jeg forstår ikke dette scenarioet. 

Hva er draft DOI? Hva er landing page? er landing page = `Public publication page for a publication`?
Dette er første gang `resource` brukes som begrep. Tidligere er det brukt `Publication`.